.github/dependabot.yml

```diff
@@ -4,6 +4,9 @@ updates:
   - package-ecosystem: "bundler"
     directory: "/"
     schedule:
-      interval: "daily"
-    allow:
-      - dependency-type: "all"
+      interval: "monthly"
+      day: "sunday"
```

Comment:
Propose to remove `day`, it's not a suitable configuration for a `monthly` interval.